docker/populate/populate.sh

```diff
@@ -0,0 +1,49 @@
+echo "Populating initial content in ${KOLIBRI_HOME}"
```

Comment:
Same thing as above. If we keep duplicating files we are going to shoot ourselves in the foot at one point.